spec/stored_queries/generic/fulltext_search.yaml

```diff
@@ -57,7 +59,7 @@ params:
 query: |
   LET search_text__norm = REGEX_REPLACE(LOWER(TRIM(@search_text)), "\\s+", " ")
   LET search_text__first_exact_tok = REGEX_SPLIT(search_text__norm, " ")[0]
-  LET search_text__icu_toks = TOKENS(@search_text, "_system::icu_tokenize")
+  LET search_text__icu_toks = TOKENS(@search_text, "icu_tokenize")
```

Comment:
Important: apparently the _system database has a small but critical amount of data on ACLs etc. so we'll just avoid touching that with an overabundance of caution

Reply:
do you want to add that somewhere in the description or as a comment?

Reply:
Adding some comments in general too